src/assets/css/librarybrowser.css

```diff
@@ -242,7 +242,7 @@
 }
 
 .mainDrawer-scrollContainer {
-    padding-bottom: 10vh;
+    margin-bottom: 10vh;
```

Comment:
Why ?

Reply:
Nevermind, saw the issue and the comment.

Reply:
I'm a git hub noob, was fixing a different issue now and was trying to make a new PR. 
Not sure how to split it into a different request. See bug 1035.

Reply:
No worries :) It's so small that we can roll it into this one.

Your issue here is that you made your original PR with the `master` branch. That's the main branch of your fork, from which you should base all your future PRs.

Once this PR is merged, you can do the following to reset your branch to the state of the `master` on this repository:

```bash
git remote add upstream https://github.com/jellyfin/jellyfin-web.git
git fetch upstream
git reset --hard upstream/master
git push origin HEAD --force-with-lease
```

This will reset your repository's master branch both locally and on Github.

Then whenever you want to make some new development, do the following to create a new branch:

```bash
git checkout master
git checkout -b <your new branch>
```

To update your master branch periodically, you can do the following;

```bash
git checkout master
git fetch upstream
git merge upstream/master --ff-only
git push origin HEAD
```

Do wait before this is merged though. Otherwise, we will lose the code in this PR.